pkg/volume/volume.go

```diff
@@ -137,6 +137,13 @@ type Detacher interface {
 	Detach() error
 }
 
+// VolumeLabeler interface provides methods to get volume labels from external cloud
+type VolumeLabeler interface {
+	// GetLabels returns labels of the volume. This function SHOULD return error
+	// when the volume does not exist.
+	GetLabels() (map[string]string, error)
```

Comment:
I really don't like how overloaded the term `Label` has become in the kubernetes ecosystem. But, alas, that ship has sailed.